verify/BUILD

```diff
@@ -0,0 +1,19 @@
+py_test(
```

Comment:
https://bazel.build/versions/master/docs/be/functions.html#licenses

It probably doesn't really matter in this repo. I think it mostly matters when you link code with more restrictive licensing rules.